### ISSUE TITLE
2019- WWW’19 - Debiasing the Human-Recommender System Feedback Loop in Collaborative Filtering

### ISSUE THREAD

**Issue by yogeswarl** [fani-lab/Adila#1]
Title: Debiasing the Human-Recommender System Feedback Loop in Collaborative Filtering 
Venue: (WWW ’19 Companion) San Francisco, CA, USA 
Year: 2019

**main problem**
Recommendation system introduce a phenomenon called as ‘filter bubble’. A situation where users only see a narrow subset of the entire range of available recommendations. Recommendation systems biggest problem is, it knows why a user likes an item, it doesn’t know why an item is not-liked by the user

**input**
A Rating matrix Ru′ i, λ, Learning rate η, MAXinteration, Iterations=0, Sizeofselection 

**Output**
Mean Absolute error, Root mean square error, Gini coefficient
The system trains the initial Matrix Factorization model and computes predictions Rˆ 

**Motivation**
Introducing debasing algorithms for Recommendation systems based on Matrix factorization during the chain of events in which users actively interact with Recommendation systems

**Previous Works and their Gaps**
-  an implicit feedback model to measure the level of confidence that a user will see an item 
- Schnabel introduced recommending items in an RS is analogous to exposing a patient to a randomized treatment in a medical study, and proposed the introduction of a propensity-scored recommendation learning system 
- Chanely et al. observed that algorithms increase homogeneity among users, thereby decreasing the utility gained by users of the system, and presented a simulation to illustrate how this effect occurs 

**Contributions**
- a unified recommendation and active learning strategy (active recommendation) during the interaction between users and the RS 
- an exposure-based collaborative filtering recommendation model that is also combined with an active recommendation to further debias the RS 
- a blind spot aware MF, which takes into account the blind spot inherent in the learning phase of the RS. 4. 

**Proposed Method**
randomly select 25 items for each of the 500 users from the completed rating matrix and start the initial training. Later on they select 20% of the ratings as the test set. This testing set is later fixed, All other ratings are considered candidate ratings: they are used to simulate the feedback loop of Recommendation system and human interaction. In each feedback loop they implement one of the algorithms to recommend items to each user to which the response is simulated

**Experiments**
An item response theory to generate a sparse rating matrix using the model proposed in the paper 'Rank aggregation via nuclear norm minimization' (https://arxiv.org/pdf/1102.4821.pdf) . 
A dataset is generate with a rating matrix R with 500 users and 500 items, total of 250,000 ratings  

**code**
https://github.com/wenlong-sun7/Debiasing-the-Human-Recommender-System-Feedback-Loop-in-Collaborative-Filtering

**Gaps of this work**
assuming that users totally agree with the recommendations in each iteration, and provide feedback.
